data/translations/zh-hans/state-data.json

```diff
@@ -3,311 +3,412 @@
     "hp_link": "https://www.sos.alabama.gov/alabama-votes",
     "registration_link": "https://www.alabamainteractive.org/sos/voter_registration/voterRegistrationWelcome.action",
     "more_info_link": "https://www.sos.alabama.gov/alabama-votes/voter/register-to-vote",
-    "ip_deadline": "选举日前 15 天",
-    "online_deadline": "选举日前 15 天",
-    "mail_postmarked_deadline": "必须在选举日前 15 天盖上邮戳",
-    "confirm_registration_link": "选举日前 30 天"
+    "ip_deadline": "2022年10月24日星期六",
+    "online_deadline": "2022年10月24日星期六",
+    "mail_postmarked_deadline": "必须在（2022年10月24日星期六）之前加盖邮戳",
```

Comment:
We need to remove the `()` from postmarked and received deadlines. We only have the translators do this so we can identify the start and end of the date in case we need to modify it later.

Reply:
fixed